web-client/src/presenter/computeds/dashboardExternalHelper.test.js

```diff
@@ -68,7 +70,22 @@ describe('petitioner dashboard helper', () => {
     expect(result.showCaseSearch).toEqual(true);
   });
 
-  it('hides case search if defined user does not have privatePractitioner or irsPractitioner role', () => {
+  it('shows case search if defined user has peititioner role', () => {
+    applicationContext.getCurrentUser = () => ({
+      role: ROLES.petitioner,
+    });
+    const result = runCompute(dashboardExternalHelper, {
+      state: {
+        closedCases: [{ something: true }],
+        openCases: [{ something: true }],
+      },
+    });
+    expect(result.showCaseList).toEqual(true);
```

Comment:
💭 In an effort to keep unit tests small and focused, does this test need all these expects? The test name indicates we are testing for the `showCaseSearch` property only

Reply:
removed all unneeded expect statements from it blocks